sample_data_config/twitter-ads-api/twitter-ads-api.config.yaml

```diff
@@ -0,0 +1,19 @@
+twitterAdsApi:
+  - dataPipelineId: twitter_ads_api_campaign_elife_journal
+    source:
+      resource: '/11/accounts/18ce53up4ko/campaigns'
+      secrets:
+        parametersFromFile:
+          - parameterName: api_key
+            filePathEnvName: TWITTER_API_KEY_FILE_PATH
```

Comment:
would include "ADS" in the env name, as you have for the config file.

Reply:
This api keys are from twitter api not just for Ads api. We can use it for twitter api too if it is needed.

Reply:
There is not specific key creation for Twitter Ads API.

Reply:
Okay, I didn't realise.